AppService.Acmebot/Startup.cs

```diff
@@ -104,6 +107,19 @@ public override void Configure(IFunctionsHostBuilder builder)
                 };
             });
 
+            builder.Services.AddSingleton(provider =>
+            {
+                var options = provider.GetRequiredService<IOptions<AcmebotOptions>>();
+                var environment = provider.GetRequiredService<AzureEnvironment>();
+
+                var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions
+                {
+                    AuthorityHost = new Uri(environment.ActiveDirectory)
+                });
+
+                return new CertificateClient(new Uri(options.Value.VaultBaseUrl), credential);
```

Comment:
I'm guessing that this should be set via an `Acmebot:VaultBaseUrl` config setting in the Function App UI in Azure ?